pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyServiceStarter.java

```diff
@@ -72,13 +72,15 @@
 
     @Parameter(names = { "-zk", "--zookeeper-servers" }, description = "Local zookeeper connection string")
     private String zookeeperServers = "";
+    @Parameter(names = { "-md", "--metadata-store" }, description = "Metadata Store service url. eg: zk:my-zk:2181")
+    private String metadataStoreUrl = "";
 
     @Deprecated
     @Parameter(names = { "-gzk", "--global-zookeeper-servers" }, description = "Global zookeeper connection string")
     private String globalZookeeperServers = "";
 
     @Parameter(names = { "-cs", "--configuration-store-servers" },
-        description = "Configuration store connection string")
+                    description = "Configuration store connection string")
     private String configurationStoreServers = "";
```

Comment:
Do we need to add `configurationMetadataStore` and deprecate this field to make it consistent with the config?

Reply:
I think the the variable name is ok, just make sure we use config.setConfigurationMetadataStoreUrl(configurationStoreServers);
instead of config.setConfigurationStoreServers(configurationStoreServers);

Reply:
Why not add `configurationMetadataStoreUrl` and add @Deprecated to this.
Also , add @Deprecated  to `zookeeperServers`.
What do you think ? @gaozhangmin @codelipenghui @RobertIndie

Reply:
> Do we need to add `configurationMetadataStore` and deprecate this field to make it consistent with the config?

+1

Reply:
done @Jason918  @RobertIndie